### PR TITLE
Allow KEL code generation in folders that are more deeply nested than…

### PR DESCRIPTION
**Pull request by gerco**
… 2 levels

### CODE REVIEW

**Comment by richardkchapman** [comment]
@GordonSmith Please review

File: comms/AttributeImpl.h [hunk 1]
```diff
@@ -15,6 +15,7 @@ class CAttributeBase : public clib::CLockableUnknown
 
 public:
     virtual IAttributeType *GetType() const = 0;
+	virtual const IModule *GetModule() const = 0;
```
Comment (richardkchapman) [inline comms/AttributeImpl.h]: Indentation - looks like you may have used tabs rather than spaces?

**Comment by gerco** [comment]
I repushed the commit with the tabs replaced by spaces.

**Comment by GordonSmith** [comment]
@gerco FYI I am hoping to test this in the next day or two.

**Comment by GordonSmith** [comment]
@gerco Is this fix for the importing of the generated ECL via .mod file (with nesting) or is it the ability to view the nested ECL after it has been imported?

Also I am assuming your testing in a "remote/legacy repository" environment?

**Comment by GordonSmith** [comment]
@richardkchapman this can be closed (for now) until we chase up:  https://track.hpccsystems.com/browse/IDE-373.

**Pull request closed without merging**